README.md

```diff
@@ -165,214 +138,19 @@ class HelloFunctionFinderProcessor : SymbolProcessor() {
-
-## Comparison to KAPT
-
-[KAPT](https://kotlinlang.org/docs/reference/kapt.html) is a remarkable
-solution which makes a large amount of Java annotation processors work
-for Kotlin programs out-of-box. The major advantages of KSP over KAPT are
-improved build performance, not tied to JVM, a more idiomatic Kotlin
-API, and the ability to understand Kotlin-only symbols.
-
-To run Java annotation processors unmodified, KAPT compiles Kotlin code
-into Java stubs that retain information that Java annotation processors
-care about. To create these stubs, KAPT needs to resolve all symbols in
-the Kotlin program. The stub generation costs roughly 1/3 of a full
-`kotlinc` analysis and the same order of `kotlinc` code-generation. For
-many annotation processors, this is much longer than the time spent in
-the processors themselves. For example, Glide looks at a very limited
-number of classes with a predefined annotation, and its code generation
-is fairly quick. Almost all of the build overhead resides in the stub
-generation phase. Switching to KSP would immediately reduce the time
-spent in the compiler by 25%.
-
-For performance evaluation, we implemented a
-[simplified version](https://github.com/google/ksp/releases/download/1.4.10-dev-experimental-20200924/miniGlide.zip)
-of [Glide](https://github.com/bumptech/glide) in KSP to make it generate code
-for the [Tachiyomi](https://github.com/inorichi/tachiyomi) project. While
-the total Kotlin compilation time of the project is 21.55 seconds on our
-test device, it took 8.67 seconds for KAPT to generate the code, and it
-took 1.15 seconds for our KSP implementation to generate the code.
-
-Unlike KAPT, processors in KSP do not see input programs from Java's point
-of view. The API is more natural to Kotlin, especially for Kotlin-specific
-features such as top-level functions. Because KSP doesn't delegate to
-`javac` like KAPT, it doesn't assume JVM-specific behaviors and can be
-used with other platforms potentially.
-
-## Limitations
-
-While KSP tries to be a simple solution for most common use cases, it has made
-several trade-offs compared to other plugin solutions. The following are not
-goals of KSP:
-
-* Examining expression-level information of source code.
-* Modifying source code.
-* 100% compatibility with the Java Annotation Processing API.
-
-We are also exploring several additional features. Note that these features are
-currently unavailable:
-
-* IDE integration: Currently IDEs know nothing about the generated code.
-* We are still investigating how to provide interoperability between processors.
-
-## For Java Annotation Processor Authors
-
-If you are familiar with the Java Annotation Processing API, see the
-[Java annotation processing to KSP reference](docs/reference.md) for how to
-implement most functions and data structures that are provided by Java's
-API.
+* [Quickstart](/docs/quickstart.md)
+* [Why use KSP?](/docs/why-ksp.md)
```

Comment:
Maybe we can put this in root as well?

Reply:
Like put the text directly in the README? Or add those as separate markdown files in the root directory?

Reply:
Yeah, I mean put text directly in README.

Reply:
The why-ksp paragraphs might be a little bit wordy so as a trade-off, would it make sense to mention the link in the introduction, in addition to being just an entry in the Resources section?

Reply:
I added links to these two resources to the opening section as well (per Ting-Yuan's suggestion)